guide/example/example-json-05.kt

```diff
@@ -4,16 +4,13 @@ package example.exampleJson05
 import kotlinx.serialization.*
 import kotlinx.serialization.json.*
 
-val format = Json { encodeDefaults = true }
```

Comment:
git's capabilities of tracking renames are quite limited and it seems like it can't handle such a chain of renames.

I see two possible solutions to keep files' git history a bit more clear:
- commit all renames separately (wow, 28 commits 😓 )
- put the new example into `example-json-32.kt` instead of 04.

WDYT?

Reply:
I'm afraid Knit functionality as a tool is quite limited in this place — it will automatically re-number all examples in a sequence (https://github.com/Kotlin/kotlinx-knit?tab=readme-ov-file#example-files)